libpathod/utils.py

```diff
@@ -3,10 +3,12 @@
 from netlib import tcp
 
 SSLVERSIONS = {
```

Comment:
Does it make sense to move mitmproxy's [`version_to_openssl`](https://github.com/mitmproxy/mitmproxy/blob/36d7e3fcd645ec9b5a7466046e044a2f1c1167ad/libmproxy/proxy/config.py#L97-L122) into netlib and re-use this?